src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -254,25 +253,25 @@ Amplify.configure(awsconfig);
 ```
 
 This will bootstrap your app to use Amplify with the deployed backend resources.
-
+ 
 **Recap:** You now have set up and configured your backend resources for Amplify authentication and are ready to start working on your front-end application.
 
 ## Determine your Auth integration path
 
 Before you implement Auth on your front-end application, you will want to evaluate the options of using the Authenticator UI component, Amplify Libraries methods, or Cognito Hosted UI for your use case. We recommend using Authenticator as the easiest to implement. You can also use Authenticator together with Amplify Libraries methods to extend capabilities when needed. Additionally, the Hosted UI is also best used with the Amplify Libraries because they provide a method to redirect to the Hosted UI.
-
+ 
```

Comment:
Controversial choice - I took out prerequisites here. Because they are making a decision but not executing the steps they do not technically need anything built for this job - the job is making a choice. If anything they need to know their Auth strategy needs that my influence their choice but technically they can make this choice without anything built yet - correct?

Reply:
We've chatted about this. We should just move pre-requisites to the top of the page.